drivers/usb/gadget/xaptum/f_psock_gadget.c

```diff
@@ -660,7 +667,14 @@ static void psock_free_instance(struct usb_function_instance *fi)
         struct f_psock_opts *psock_opts;
 
         psock_opts = container_of(fi, struct f_psock_opts, func_inst);
-        kfree(psock_opts);
```

Comment:
Ditto here.  `psock_opts` should never return null --- it's just a simple lookup via `container_of`.  If it does, there's a bug where the these structs are initialized. Let's fix it there, rather than masking the bug here.

Reply:
I don't think there was ever a memory leak here, this is probably just a test I created and lost track of.